lib/pbench/server/api/resources/query_apis/metadata_index/index_mappings.py

```diff
@@ -85,4 +91,4 @@ def get(self, index_name: AnyStr) -> Dict[AnyStr, Any]:
             self.logger.exception(
                 "Document template {} not found in the database.", index_name
             )
-            abort(HTTPStatus.NOT_FOUND, message="Mapping not found")
+            abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
```

Comment:
I've been wondering whether we should change all of these (on `Template` lookup failures) to 503/service-unavailable since they'll be resolved when we index our first tarball. (Though that's also my main motivation for thinking we ought to pre-load the mapping templates at startup rather than waiting for the first index operation, which might make this irrelevant.)

Reply:
I concur that the server shouldn't fail due to the template mapping being unavailable, and so, if it does, then it's not a matter of "not found" but rather an internal error (which I might be convinced should be reported as a 503 although I'm not "there" yet).

As for whether the server code should depend on the database being pre-populated during installation or not, I feel like that's going to be inconvenient for development (I think it would be better if the server code just "fixed" the situation when it encounters it).  But, perhaps this isn't the place for that discussion.